### DISCUSSION TITLE
Effect of food on gastric emptying time

### DISCUSSION THREAD

**Discussion by MCR15** [Open-Systems-Pharmacology/Forum#1693] [Questions & Problems]
I need to track the GET variation over time in the simulation, how can I obtain this? I have tried creating an observer, but in the obtained graph the effect of food administration on GET is not reflected; it only indicates the predefined value (15 minutes), even though the simulated plasma concentrations do capture the effect of food.
![Picture1](https://user-images.githubusercontent.com/155985857/295577463-e3be06e5-0d70-41ae-bc83-9960ffa97a2b.png)

**Comment by PavelBal** [comment]
The parameter that is modified after meal intake is `Organism|Lumen|Stomach|Intestinal transit rate (absolute)".

However, I am afraid that you can only plot this parameter in MoBi and not in PK-Sim.

**Comment by MCR15** [comment]
Thank you for your answer Pavel. 
I have been trying to plot the GET and the intestinal transit rate in MoBi, but the effect of food is still not observed, it seems i am missing something, is there a guide to do it in MoBi?
![image](https://github.com/Open-Systems-Pharmacology/Forum/assets/155985857/8ff52f56-90a2-4fe1-a89a-931733a8fb05)


I need to plot the variation in GET or the intestinal transit rate vs time after a meal intake, something like the plot in this comment https://github.com/Open-Systems-Pharmacology/Forum/issues/461#issuecomment-586755126

**Reply by PavelBal** [reply]
If I understand your second screenshot correctly, you are still plotting `1/GET` and not the newly added parameter "Intestinal transit rate".

**Reply by MCR15** [reply]
Thank you for your help!
